docs/commands/index.md

```diff
@@ -15,6 +15,7 @@ Redocly CLI currently supports the following commands:
 * [push](push.md)
 * [split](split.md)
 * [stats](stats.md)
+* [build-docs](build-docs.md)
```

Comment:
This shouldn't go here. These are in alphabetical order.

Reply:
fixed